ssp

```diff
@@ -6361,7 +6449,7 @@ sub check_for_usr_local_include_jpeglib_h {
 }
 
 sub check_for_bw_module_and_more_than_1024_vhosts {
-    my $httpdconf = '/usr/local/apache/conf/httpd.conf';
+    my $httpdconf = $EA4 ? '/etc/apache2/conf/httpd.conf' : '/usr/local/apache/conf/httpd.conf';
```

Comment:
In either case /usr/local/apache/conf/httpd.conf will be a valid path here, right?  I suppose this is a better path than changing the -f on line 6365/6453 to a -e, though.  OK